package.json

```diff
@@ -9,7 +9,9 @@
     "lint": "eslint --ext .js,jsx ./src",
     "test": "razzle test --env=jsdom",
     "start": "NODE_ENV=production node build/server.js",
-    "speculate": "speculate"
+    "speculate": "speculate",
+    "test:e2e": "npx cypress run",
+    "test:e2e:debug": "npx cypress open"
```

Comment:
It would be more specific to name this `test:e2e:interactive`, it might not always be for debugging reasons that this is ran.

Reply:
👌